src/bundle/Resources/config/routing.yml

```diff
@@ -570,6 +570,13 @@ ezplatform.content.translate:
         _controller: 'EzPlatformAdminUiBundle:ContentEdit:translate'
         fromLanguageCode: ~
 
+ibexa.content.translate_with_location:
+    path: /content/{contentId}/location/{locationId}/translate/{toLanguageCode}/{fromLanguageCode}
+    methods: ['GET', 'POST']
+    defaults:
+        _controller: 'EzPlatformAdminUiBundle:ContentEdit:translate'
```

Comment:
Isn't this a legacy way of referencing controllers?

Using FQCN is recommended afaik.